[2025-04-01T03:26:44.330+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-01T03:26:44.348+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_dag_with_python_operator_v01.greet manual__2025-04-01T03:26:43.623806+00:00 [queued]>
[2025-04-01T03:26:44.356+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_dag_with_python_operator_v01.greet manual__2025-04-01T03:26:43.623806+00:00 [queued]>
[2025-04-01T03:26:44.357+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-01T03:26:44.370+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): greet> on 2025-04-01 03:26:43.623806+00:00
[2025-04-01T03:26:44.377+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=23929) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-01T03:26:44.378+0000] {standard_task_runner.py:72} INFO - Started process 23930 to run task
[2025-04-01T03:26:44.377+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_dag_with_python_operator_v01', 'greet', 'manual__2025-04-01T03:26:43.623806+00:00', '--job-id', '112', '--raw', '--subdir', 'DAGS_FOLDER/create_dag_with_python_operator.py', '--cfg-path', '/tmp/tmps3rlog7u']
[2025-04-01T03:26:44.379+0000] {standard_task_runner.py:105} INFO - Job 112: Subtask greet
[2025-04-01T03:26:44.424+0000] {task_command.py:467} INFO - Running <TaskInstance: our_dag_with_python_operator_v01.greet manual__2025-04-01T03:26:43.623806+00:00 [running]> on host 9663692256f6
[2025-04-01T03:26:44.503+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Rajesh' AIRFLOW_CTX_DAG_ID='our_dag_with_python_operator_v01' AIRFLOW_CTX_TASK_ID='greet' AIRFLOW_CTX_EXECUTION_DATE='2025-04-01T03:26:43.623806+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-01T03:26:43.623806+00:00'
[2025-04-01T03:26:44.504+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-01T03:26:44.518+0000] {logging_mixin.py:190} INFO - Hello World
[2025-04-01T03:26:44.519+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-01T03:26:44.527+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-01T03:26:44.528+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=our_dag_with_python_operator_v01, task_id=greet, run_id=manual__2025-04-01T03:26:43.623806+00:00, execution_date=20250401T032643, start_date=20250401T032644, end_date=20250401T032644
[2025-04-01T03:26:44.563+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-01T03:26:44.579+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-01T03:26:44.581+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
